File: Resueltos_por_competidor/Thomas_Molina/Dominopiling.java
/*
 * Autor: Thomas Molina
 * Problema: A. Domino piling
 * Juez online: Codeforces
 * Veredicto: Accepted
 * URL: https://codeforces.com/problemset/problem/50/A
 */


import java.util.Scanner;

public class Dominopiling {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int m = sc.nextInt();
		int n = sc.nextInt();
		
		int r = (m*n)/2;
		
		System.out.println(r);
	}

}

File: Resueltos_por_competidor/Carlos_Cobaleda/Domino_piling.cpp
/*   
Autor: Carlos Eduardo Cobaleda Moreno
Problema: Domino piling
Juez online: Codeforces
Veredicto: Accepted
URL: https://codeforces.com/contest/50/problem/A
*/

#include <bits/stdc++.h>
#include <cmath>
using namespace std;
 
int main() {
    int a,b; cin >> a >> b;
    double r = (a*b) / 2;
 
    cout << floor(r) << endl;
    return 0;
}
